tests/remote_resources/resources.xml

```diff
@@ -0,0 +1,8 @@
+<resources>
```

Comment:
Please move this file to the xmos_test_resources repo. This will allow these test resources to be shared by other libraries tests.